desktop/components/main_layout/templates/error.jade

```diff
@@ -1,27 +1,26 @@
-extends ./index
+doctype html
```

Comment:
I was starting to address this issue https://github.com/artsy/force/issues/1148 but I should probably leave that for another PR—will update.